docs/docs/azure.md

```diff
@@ -38,132 +38,21 @@ question mark is the size. Other distributions should work as long as they are o
-
-DCAP is designed to work on many different server topologies. So, rather than directly connecting to Intel services to retrieve this information, the cloud vendor or the owner of the SGX system must provide a DCAP client plugin with the required information. Intel provides a generic DCAP client plugin as part of the DCAP runtime. To use this, you also need to set up a Provisioning Certificate Caching Service (PCCS). Intel provides an example and some instructions [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs/README.md). If you would like to use Intel's reference implementation of their [PCCS service](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/pccs), then additional work may be required to provide data in the correct format for Conclave. Please contact R3 support if you need to set this up.
-
-Microsoft has written a DCAP client plugin that works with its Azure Confidential Compute virtual machines. In fact, it also works outside of Azure for single CPU systems, but this may not always be the case.
-
-Conclave's bundled Azure client plugin will *only* be used if no other plugin has been found on the system.
-The runtime will use the first `.so` it encounters in the search order below:
-```
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
-/usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
-/usr/lib/libdcap_quoteprov.so.1
-/usr/lib/libdcap_quoteprov.so
-```
-Should you decide to use a bundled version (recommended), ensure the files listed above don't exist (delete or rename
-them if necessary).
-
-!!! important
-    The bundled DCAP Azure client plugin works with all current generation Azure confidential VMs, as well as existing
-    CPUs. However, if Azure extends their offerings to include new platforms, or if you use a new generation
-    system, then the bundled Azure client plugin may throw an error
-    when attempting to load your enclave. In this case, [follow the instructions below](#azure-client-plugin) to
-    build/install an updated version of the Azure client plugin that supports the platform.
-
-You may want to set the Azure DCAP client logging level to FATAL as the default setting is quite verbose:
-```sh
-export AZDCAP_DEBUG_LOG_LEVEL=FATAL
-```
-
-If you are using the bundled version, you can skip the rest of this section. Otherwise, read on for instructions on how to manually configure the DCAP plugin.
-
-#### Azure client plugin
+You have set up an Azure VM with SGX.
 
-* Identify the currently installed DCAP client plugin. It will always have one of the following names: `libdcap_quoteprov.so.1` or `libdcap_quoteprov.so`. You might find other similarly named files, but they won't be used as a plugin.
-```sh
-ls /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so*
-ls /usr/lib/libdcap_quoteprov.so*
-```
+You need to use the DCAP protocol for attestation. For this, you need to use the `AttestationParameters.DCAP()` class 
```

Comment:
Can you reference the class in the API docs.

Reply:
Done.